大神，比较紧急，wndr3800这两天都编译出错，提示wnr8500什么什么，请看以下日志。

反馈bug/问题模板，提建议请删除

## 1.关于你要提交的问题

Q：是否搜索了issue (使用 "x" 选择)
* [x] 没有类似的issue

## 2. 详细叙述

### (1) 具体问题

A：编译出错

### (2) 路由器型号和固件版本

A：网件 wndr3800

### (3) 详细日志
A：
2020-05-24T11:23:28.1680535Z make[4]: Entering directory '/workdir/openwrt/target/linux/ar71xx'
2020-05-24T11:23:28.2453059Z rm -f /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/localversion
2020-05-24T11:23:28.2474762Z /workdir/openwrt/scripts/kconfig.pl  + + /workdir/openwrt/target/linux/generic/config-4.14 /workdir/openwrt/target/linux/ar71xx/config-4.14 /workdir/openwrt/target/linux/ar71xx/generic/config-default > /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target
2020-05-24T11:23:28.2952618Z awk '/^(#[[:space:]]+)?CONFIG_KERNEL/{sub("CONFIG_KERNEL_","CONFIG_");print}' /workdir/openwrt/.config >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target
2020-05-24T11:23:28.3027138Z echo "# CONFIG_KALLSYMS_EXTRA_PASS is not set" >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target
2020-05-24T11:23:28.3047645Z echo "# CONFIG_KALLSYMS_ALL is not set" >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target
2020-05-24T11:23:28.3066626Z echo "CONFIG_KALLSYMS_UNCOMPRESSED=y" >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target
2020-05-24T11:23:28.3082747Z /workdir/openwrt/scripts/package-metadata.pl kconfig /workdir/openwrt/tmp/.packageinfo /workdir/openwrt/.config 4.14 > /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.override
2020-05-24T11:23:28.3995936Z /workdir/openwrt/scripts/kconfig.pl 'm+' '+' /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.target /dev/null /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.override > /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set
2020-05-24T11:23:28.4488413Z mv /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.old
2020-05-24T11:23:28.4517547Z grep -v INITRAMFS /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.old > /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set
2020-05-24T11:23:28.4591573Z echo 'CONFIG_INITRAMFS_SOURCE=""' >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set
2020-05-24T11:23:28.4610101Z echo '# CONFIG_INITRAMFS_FORCE is not set' >> /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set
2020-05-24T11:23:28.4628483Z rm -rf /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/modules
2020-05-24T11:23:28.4691214Z cmp -s /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.prev || { cp /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config; cp /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.prev; }
2020-05-24T11:23:28.4719654Z export MAKEFLAGS= ; [ -d /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/user_headers ] || make -C /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180 KCFLAGS="-iremap/workdir/openwrt/build_dir/target-mips_24kc_musl:target-mips_24kc_musl" HOSTCFLAGS="-O2 -I/workdir/openwrt/staging_dir/host/include  -Wall -Wmissing-prototypes -Wstrict-prototypes" CROSS_COMPILE="mips-openwrt-linux-musl-" ARCH="mips" KBUILD_HAVE_NLS=no KBUILD_BUILD_USER="" KBUILD_BUILD_HOST="" KBUILD_BUILD_TIMESTAMP="Sun May 24 05:05:57 2020" KBUILD_BUILD_VERSION="0" HOST_LOADLIBES="-L/workdir/openwrt/staging_dir/host/lib" KBUILD_HOSTLDLIBS="-L/workdir/openwrt/staging_dir/host/lib" CONFIG_SHELL="bash" V=''  cmd_syscalls= KERNELRELEASE=4.14.180 CC="mips-openwrt-linux-musl-gcc" INSTALL_HDR_PATH=/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/user_headers headers_install
2020-05-24T11:23:28.4739847Z grep '=[ym]' /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.config.set | LC_ALL=C sort | mkhash md5 > /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.vermagic
2020-05-24T11:23:28.4784879Z touch /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.configured
2020-05-24T11:23:28.4809795Z rm -f /workdir/openwrt/build_dir/target-mips_24kc_musl/root-ar71xx/init
2020-05-24T11:23:28.4835156Z make -C /workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180 KCFLAGS="-iremap/workdir/openwrt/build_dir/target-mips_24kc_musl:target-mips_24kc_musl" HOSTCFLAGS="-O2 -I/workdir/openwrt/staging_dir/host/include  -Wall -Wmissing-prototypes -Wstrict-prototypes" CROSS_COMPILE="mips-openwrt-linux-musl-" ARCH="mips" KBUILD_HAVE_NLS=no KBUILD_BUILD_USER="" KBUILD_BUILD_HOST="" KBUILD_BUILD_TIMESTAMP="Sun May 24 05:05:57 2020" KBUILD_BUILD_VERSION="0" HOST_LOADLIBES="-L/workdir/openwrt/staging_dir/host/lib" KBUILD_HOSTLDLIBS="-L/workdir/openwrt/staging_dir/host/lib" CONFIG_SHELL="bash" V=''  cmd_syscalls= KERNELRELEASE=4.14.180 CC="mips-openwrt-linux-musl-gcc" all modules
2020-05-24T11:23:28.4863573Z make[5]: Entering directory '/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180'
2020-05-24T11:23:29.4038536Z   CHK     include/config/kernel.release
2020-05-24T11:23:29.4598858Z   CHK     include/generated/uapi/linux/version.h
2020-05-24T11:23:29.4730167Z   CHK     include/generated/utsrelease.h
2020-05-24T11:23:29.5151502Z   CHK     include/generated/bounds.h
2020-05-24T11:23:29.5206322Z   CHK     include/generated/timeconst.h
2020-05-24T11:23:29.5332988Z   CHK     include/generated/asm-offsets.h
2020-05-24T11:23:29.5424270Z   CALL    scripts/checksyscalls.sh
2020-05-24T11:23:29.5668999Z   CHK     scripts/mod/devicetable-offsets.h
2020-05-24T11:23:29.6049498Z   CHK     include/generated/compile.h
2020-05-24T11:23:30.1943123Z   CC      arch/mips/ath79/mach-tl-wdr8500-v1.o
2020-05-24T11:23:30.4463082Z arch/mips/ath79/mach-tl-wdr8500-v1.c:108:4: error: 'struct mdio_board_info' has no member named 'phy_addr'; did you mean 'mdio_addr'?
2020-05-24T11:23:30.4464027Z    .phy_addr = 0,
2020-05-24T11:23:30.4464791Z     ^~~~~~~~
2020-05-24T11:23:30.4464913Z     mdio_addr
2020-05-24T11:23:30.4465657Z arch/mips/ath79/mach-tl-wdr8500-v1.c:105:60: error: missing braces around initializer [-Werror=missing-braces]
2020-05-24T11:23:30.4465803Z  static struct mdio_board_info TL_WDR8500_v1_mdio0_info[] = {
2020-05-24T11:23:30.4465926Z                                                             ^
2020-05-24T11:23:30.4466203Z arch/mips/ath79/mach-tl-wdr8500-v1.c:108:15:
2020-05-24T11:23:30.4466294Z    .phy_addr = 0,
2020-05-24T11:23:30.4466398Z                {}
2020-05-24T11:23:30.4466734Z arch/mips/ath79/mach-tl-wdr8500-v1.c:105:60: error: missing braces around initializer [-Werror=missing-braces]
2020-05-24T11:23:30.4466888Z  static struct mdio_board_info TL_WDR8500_v1_mdio0_info[] = {
2020-05-24T11:23:30.4466985Z                                                             ^
2020-05-24T11:23:30.4467260Z arch/mips/ath79/mach-tl-wdr8500-v1.c:108:15:
2020-05-24T11:23:30.4467371Z    .phy_addr = 0,
2020-05-24T11:23:30.4467477Z                {}
2020-05-24T11:23:30.4506326Z cc1: all warnings being treated as errors
2020-05-24T11:23:30.4536326Z make[7]: *** [arch/mips/ath79/mach-tl-wdr8500-v1.o] Error 1
2020-05-24T11:23:30.4536757Z scripts/Makefile.build:326: recipe for target 'arch/mips/ath79/mach-tl-wdr8500-v1.o' failed
2020-05-24T11:23:30.4544787Z make[6]: *** [arch/mips/ath79] Error 2
2020-05-24T11:23:30.4545510Z scripts/Makefile.build:585: recipe for target 'arch/mips/ath79' failed
2020-05-24T11:23:30.4547093Z make[5]: *** [arch/mips] Error 2
2020-05-24T11:23:30.4547549Z Makefile:1052: recipe for target 'arch/mips' failed
2020-05-24T11:23:30.4548237Z make[5]: Leaving directory '/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180'
2020-05-24T11:23:30.4549887Z make[4]: *** [/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.image] Error 2
2020-05-24T11:23:30.4550300Z Makefile:24: recipe for target '/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.image' failed
2020-05-24T11:23:30.4552102Z make[3]: *** [install] Error 2
2020-05-24T11:23:30.4552443Z make[4]: Leaving directory '/workdir/openwrt/target/linux/ar71xx'
2020-05-24T11:23:30.4553407Z Makefile:13: recipe for target 'install' failed
2020-05-24T11:23:30.4553710Z make[3]: Leaving directory '/workdir/openwrt/target/linux'
2020-05-24T11:23:30.4554444Z time: target/linux/install#1.70#0.68#2.31
2020-05-24T11:23:30.4559450Z make[2]: *** [target/linux/install] Error 2
2020-05-24T11:23:30.4559852Z target/Makefile:23: recipe for target 'target/linux/install' failed
2020-05-24T11:23:30.4567036Z make[2]: Leaving directory '/workdir/openwrt'
2020-05-24T11:23:30.4579967Z make[1]: *** [/workdir/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install] Error 2
2020-05-24T11:23:30.4580378Z target/Makefile:19: recipe for target '/workdir/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install' failed
2020-05-24T11:23:30.4586845Z make[1]: Leaving directory '/workdir/openwrt'
2020-05-24T11:23:30.4601516Z make: *** [world] Error 2
2020-05-24T11:23:30.4601992Z /workdir/openwrt/include/toplevel.mk:225: recipe for target 'world' failed
2020-05-24T11:23:30.4679438Z ##[error]Process completed with exit code 2.
2020-05-24T11:23:30.4859893Z ##[group]Run df -hT
2020-05-24T11:23:30.4860134Z [36;1mdf -hT[0m
2020-05-24T11:23:30.4906099Z shell: /bin/bash -e {0}
2020-05-24T11:23:30.4906227Z env:
2020-05-24T11:23:30.4906392Z   REPO_URL: https://github.com/coolsnowwolf/lede
2020-05-24T11:23:30.4906516Z   REPO_BRANCH: master
2020-05-24T11:23:30.4906605Z   FEEDS_CONF: feeds.conf.default
2020-05-24T11:23:30.4906708Z   CONFIG_FILE: .config
2020-05-24T11:23:30.4906819Z   DIY_P1_SH: diy-part1.sh
2020-05-24T11:23:30.4906919Z   DIY_P2_SH: diy-part2.sh
2020-05-24T11:23:30.4907005Z   SSH_ACTIONS: false
2020-05-24T11:23:30.4907103Z   UPLOAD_BIN_DIR: false
2020-05-24T11:23:30.4907202Z   UPLOAD_FIRMWARE: true
2020-05-24T11:23:30.4907301Z   UPLOAD_COWTRANSFER: false
2020-05-24T11:23:30.4907387Z   UPLOAD_WETRANSFER: false
2020-05-24T11:23:30.4907487Z   TZ: Asia/Shanghai
2020-05-24T11:23:30.4907588Z ##[endgroup]
2020-05-24T11:23:30.4999175Z Filesystem     Type      Size  Used Avail Use% Mounted on
2020-05-24T11:23:30.5000202Z udev           devtmpfs  3.4G     0  3.4G   0% /dev
2020-05-24T11:23:30.5000360Z tmpfs          tmpfs     693M  952K  692M   1% /run
2020-05-24T11:23:30.5000459Z /dev/sda1      ext4       84G   39G   45G  47% /
2020-05-24T11:23:30.5000572Z tmpfs          tmpfs     3.4G  8.0K  3.4G   1% /dev/shm
2020-05-24T11:23:30.5000686Z tmpfs          tmpfs     5.0M     0  5.0M   0% /run/lock
2020-05-24T11:23:30.5000797Z tmpfs          tmpfs     3.4G     0  3.4G   0% /sys/fs/cgroup
2020-05-24T11:23:30.5000909Z /dev/loop0     squashfs   94M   94M     0 100% /snap/core/9066
2020-05-24T11:23:30.5001027Z /dev/loop1     squashfs   40M   40M     0 100% /snap/hub/43
2020-05-24T11:23:30.5001144Z /dev/sda15     vfat      105M  3.6M  101M   4% /boot/efi
2020-05-24T11:23:30.5001254Z /dev/sdb1      ext4       14G   41M   13G   1% /mnt
2020-05-24T11:23:30.5035034Z ##[group]Run cd openwrt/bin/targets/*/*
2020-05-24T11:23:30.5035244Z [36;1mcd openwrt/bin/targets/*/*[0m
2020-05-24T11:23:30.5035335Z [36;1mrm -rf packages[0m
2020-05-24T11:23:30.5035443Z [36;1mecho "::set-env name=FIRMWARE::$PWD"[0m
2020-05-24T11:23:30.5035552Z [36;1mecho "::set-output name=status::success"[0m
2020-05-24T11:23:30.5082974Z shell: /bin/bash -e {0}
2020-05-24T11:23:30.5083076Z env:
2020-05-24T11:23:30.5083356Z   REPO_URL: https://github.com/coolsnowwolf/lede
2020-05-24T11:23:30.5083481Z   REPO_BRANCH: master
2020-05-24T11:23:30.5083583Z   FEEDS_CONF: feeds.conf.default
2020-05-24T11:23:30.5083682Z   CONFIG_FILE: .config
2020-05-24T11:23:30.5083782Z   DIY_P1_SH: diy-part1.sh
2020-05-24T11:23:30.5083888Z   DIY_P2_SH: diy-part2.sh
2020-05-24T11:23:30.5083989Z   SSH_ACTIONS: false
2020-05-24T11:23:30.5084074Z   UPLOAD_BIN_DIR: false
2020-05-24T11:23:30.5084172Z   UPLOAD_FIRMWARE: true
2020-05-24T11:23:30.5084270Z   UPLOAD_COWTRANSFER: false
2020-05-24T11:23:30.5084387Z   UPLOAD_WETRANSFER: false
2020-05-24T11:23:30.5084491Z   TZ: Asia/Shanghai
2020-05-24T11:23:30.5084593Z ##[endgroup]
2020-05-24T11:23:30.5321223Z ##[group]Run actions/upload-artifact@master
2020-05-24T11:23:30.5321345Z with:
2020-05-24T11:23:30.5321446Z   name: OpenWrt_firmware
2020-05-24T11:23:30.5321563Z   path: /home/runner/work/LEDE-wndr3800/LEDE-wndr3800/openwrt/bin/targets/ar71xx/generic
2020-05-24T11:23:30.5321669Z env:
2020-05-24T11:23:30.5321818Z   REPO_URL: https://github.com/coolsnowwolf/lede
2020-05-24T11:23:30.5321938Z   REPO_BRANCH: master
2020-05-24T11:23:30.5322038Z   FEEDS_CONF: feeds.conf.default
2020-05-24T11:23:30.5322246Z   CONFIG_FILE: .config
2020-05-24T11:23:30.5322344Z   DIY_P1_SH: diy-part1.sh
2020-05-24T11:23:30.5322444Z   DIY_P2_SH: diy-part2.sh
2020-05-24T11:23:30.5322543Z   SSH_ACTIONS: false
2020-05-24T11:23:30.5322623Z   UPLOAD_BIN_DIR: false
2020-05-24T11:23:30.5322719Z   UPLOAD_FIRMWARE: true
2020-05-24T11:23:30.5322817Z   UPLOAD_COWTRANSFER: false
2020-05-24T11:23:30.5322914Z   UPLOAD_WETRANSFER: false
2020-05-24T11:23:30.5322998Z   TZ: Asia/Shanghai
2020-05-24T11:23:30.5323139Z   FIRMWARE: /home/runner/work/LEDE-wndr3800/LEDE-wndr3800/openwrt/bin/targets/ar71xx/generic
2020-05-24T11:23:30.5323245Z ##[endgroup]
2020-05-24T11:23:30.9392683Z With the provided path, there will be 1 files uploaded
2020-05-24T11:23:31.1548143Z Total size of all the files uploaded is 844 bytes
2020-05-24T11:23:31.1913973Z Finished uploading artifact OpenWrt_firmware. Reported size is 844 bytes. There were 0 items that failed to upload
2020-05-24T11:23:31.1916248Z Artifact OpenWrt_firmware has been successfully uploaded!
2020-05-24T11:23:31.2036442Z Post job cleanup.
2020-05-24T11:23:31.3289810Z [command]/usr/bin/git version
2020-05-24T11:23:31.3354280Z git version 2.26.2
2020-05-24T11:23:31.3454772Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-05-24T11:23:31.4006469Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-05-24T11:23:31.4088649Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-05-24T11:23:31.4089263Z http.https://github.com/.extraheader
2020-05-24T11:23:31.4111314Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-05-24T11:23:31.4164731Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-05-24T11:23:31.4545115Z Cleaning up orphan processes

日志连接：
https://pipelines.actions.githubusercontent.com/Ahyzfnr5zkWXCar8A2zvqlyNbviMogNmY26e8Z6yMEkkUOZP1c/_apis/pipelines/1/runs/1/signedlogcontent/3?urlExpires=2020-05-24T13%3A53%3A47.5131568Z&urlSigningMethod=HMACV1&urlSignature=IsPJPeLzanF%2FAnMB%2BJ1PAYSmXeNaiZQGmRf6LziNPDQ%3D

> > 你好，请问是在#include <linux/gpio.h>下面增加#include <linux/phy.h>这一行是吗？请问有同有办法在github上编译时更改呢，自己的配置太慢了，甚至慢于云编译。
> 
> 要不你等我的pr merge了再试试看吧
以下是刚刚编译时的日志，已经修改过那个文件了，看起来错误是一样的。
2020-05-25T04:20:05.4400910Z arch/mips/ath79/mach-tl-wdr8500-v1.c:109:15:
2020-05-25T04:20:05.4401007Z    .phy_addr = 0,
2020-05-25T04:20:05.4401113Z                {}
2020-05-25T04:20:05.4441207Z cc1: all warnings being treated as errors
2020-05-25T04:20:05.4467950Z make[7]: *** [arch/mips/ath79/mach-tl-wdr8500-v1.o] Error 1
2020-05-25T04:20:05.4468351Z scripts/Makefile.build:326: recipe for target 'arch/mips/ath79/mach-tl-wdr8500-v1.o' failed
2020-05-25T04:20:05.4477919Z scripts/Makefile.build:585: recipe for target 'arch/mips/ath79' failed
2020-05-25T04:20:05.4478911Z make[6]: *** [arch/mips/ath79] Error 2
2020-05-25T04:20:05.4482756Z make[5]: *** [arch/mips] Error 2
2020-05-25T04:20:05.4483194Z Makefile:1052: recipe for target 'arch/mips' failed
2020-05-25T04:20:05.4483772Z make[4]: *** [/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.image] Error 2
2020-05-25T04:20:05.4487426Z make[3]: *** [install] Error 2
2020-05-25T04:20:05.4487936Z make[5]: Leaving directory '/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180'
2020-05-25T04:20:05.4494663Z Makefile:24: recipe for target '/workdir/openwrt/build_dir/target-mips_24kc_musl/linux-ar71xx_generic/linux-4.14.180/.image' failed
2020-05-25T04:20:05.4495560Z make[4]: Leaving directory '/workdir/openwrt/target/linux/ar71xx'
2020-05-25T04:20:05.4496009Z Makefile:13: recipe for target 'install' failed
2020-05-25T04:20:05.4496442Z make[3]: Leaving directory '/workdir/openwrt/target/linux'
2020-05-25T04:20:05.4496670Z time: target/linux/install#1.59#0.64#2.16
2020-05-25T04:20:05.4497064Z target/Makefile:23: recipe for target 'target/linux/install' failed
2020-05-25T04:20:05.4514695Z make[2]: *** [target/linux/install] Error 2
2020-05-25T04:20:05.4515121Z make[2]: Leaving directory '/workdir/openwrt'
2020-05-25T04:20:05.4515702Z make[1]: *** [/workdir/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install] Error 2
2020-05-25T04:20:05.4517896Z target/Makefile:19: recipe for target '/workdir/openwrt/staging_dir/target-mips_24kc_musl/stamp/.target_install' failed
2020-05-25T04:20:05.4523621Z make[1]: Leaving directory '/workdir/openwrt'
2020-05-25T04:20:05.4540434Z make: *** [world] Error 2
2020-05-25T04:20:05.4540918Z /workdir/openwrt/include/toplevel.mk:225: recipe for target 'world' failed
2020-05-25T04:20:05.4713938Z ##[error]Process completed with exit code 2.
2020-05-25T04:20:05.5245820Z ##[group]Run df -hT
2020-05-25T04:20:05.5246028Z [36;1mdf -hT[0m
2020-05-25T04:20:05.5297226Z shell: /bin/bash -e {0}
2020-05-25T04:20:05.5297332Z env:
2020-05-25T04:20:05.5297507Z   REPO_URL: https://github.com/Mr-hsing/lede
2020-05-25T04:20:05.5297634Z   REPO_BRANCH: master
2020-05-25T04:20:05.5297741Z   FEEDS_CONF: feeds.conf.default
2020-05-25T04:20:05.5297850Z   CONFIG_FILE: .config
2020-05-25T04:20:05.5297951Z   DIY_P1_SH: diy-part1.sh
2020-05-25T04:20:05.5298058Z   DIY_P2_SH: diy-part2.sh
2020-05-25T04:20:05.5298162Z   SSH_ACTIONS: false
2020-05-25T04:20:05.5298266Z   UPLOAD_BIN_DIR: false
2020-05-25T04:20:05.5298366Z   UPLOAD_FIRMWARE: true
2020-05-25T04:20:05.5298470Z   UPLOAD_COWTRANSFER: false
2020-05-25T04:20:05.5298575Z   UPLOAD_WETRANSFER: false
2020-05-25T04:20:05.5298680Z   TZ: Asia/Shanghai
2020-05-25T04:20:05.5298771Z ##[endgroup]
2020-05-25T04:20:05.5642133Z Filesystem     Type      Size  Used Avail Use% Mounted on
2020-05-25T04:20:05.5642830Z udev           devtmpfs  3.4G     0  3.4G   0% /dev
2020-05-25T04:20:05.5642982Z tmpfs          tmpfs     693M  952K  692M   1% /run
2020-05-25T04:20:05.5643134Z /dev/sdb1      ext4       84G   39G   45G  47% /
2020-05-25T04:20:05.5643291Z tmpfs          tmpfs     3.4G  8.0K  3.4G   1% /dev/shm
2020-05-25T04:20:05.5643448Z tmpfs          tmpfs     5.0M     0  5.0M   0% /run/lock
2020-05-25T04:20:05.5643589Z tmpfs          tmpfs     3.4G     0  3.4G   0% /sys/fs/cgroup
2020-05-25T04:20:05.5643749Z /dev/loop0     squashfs   94M   94M     0 100% /snap/core/9066
2020-05-25T04:20:05.5643907Z /dev/loop1     squashfs   40M   40M     0 100% /snap/hub/43
2020-05-25T04:20:05.5644299Z /dev/sdb15     vfat      105M  3.6M  101M   4% /boot/efi
2020-05-25T04:20:05.5644438Z /dev/sda1      ext4       14G   41M   13G   1% /mnt
2020-05-25T04:20:05.5675273Z ##[group]Run cd openwrt/bin/targets/*/*
2020-05-25T04:20:05.5675519Z [36;1mcd openwrt/bin/targets/*/*[0m
2020-05-25T04:20:05.5675661Z [36;1mrm -rf packages[0m
2020-05-25T04:20:05.5675788Z [36;1mecho "::set-env name=FIRMWARE::$PWD"[0m
2020-05-25T04:20:05.5675933Z [36;1mecho "::set-output name=status::success"[0m
2020-05-25T04:20:05.5727746Z shell: /bin/bash -e {0}
2020-05-25T04:20:05.5727847Z env:
2020-05-25T04:20:05.5728016Z   REPO_URL: https://github.com/Mr-hsing/lede
2020-05-25T04:20:05.5728143Z   REPO_BRANCH: master
2020-05-25T04:20:05.5728253Z   FEEDS_CONF: feeds.conf.default
2020-05-25T04:20:05.5728344Z   CONFIG_FILE: .config
2020-05-25T04:20:05.5728449Z   DIY_P1_SH: diy-part1.sh
2020-05-25T04:20:05.5728557Z   DIY_P2_SH: diy-part2.sh
2020-05-25T04:20:05.5728663Z   SSH_ACTIONS: false
2020-05-25T04:20:05.5728764Z   UPLOAD_BIN_DIR: false
2020-05-25T04:20:05.5728871Z   UPLOAD_FIRMWARE: true
2020-05-25T04:20:05.5728978Z   UPLOAD_COWTRANSFER: false
2020-05-25T04:20:05.5729099Z   UPLOAD_WETRANSFER: false
2020-05-25T04:20:05.5729208Z   TZ: Asia/Shanghai
2020-05-25T04:20:05.5729316Z ##[endgroup]
2020-05-25T04:20:05.5973654Z ##[group]Run actions/upload-artifact@master
2020-05-25T04:20:05.5973767Z with:
2020-05-25T04:20:05.5973877Z   name: OpenWrt_firmware
2020-05-25T04:20:05.5974000Z   path: /home/runner/work/3800test2/3800test2/openwrt/bin/targets/ar71xx/generic
2020-05-25T04:20:05.5974125Z env:
2020-05-25T04:20:05.5974267Z   REPO_URL: https://github.com/Mr-hsing/lede
2020-05-25T04:20:05.5974393Z   REPO_BRANCH: master
2020-05-25T04:20:05.5974496Z   FEEDS_CONF: feeds.conf.default
2020-05-25T04:20:05.5974603Z   CONFIG_FILE: .config
2020-05-25T04:20:05.5974691Z   DIY_P1_SH: diy-part1.sh
2020-05-25T04:20:05.5974795Z   DIY_P2_SH: diy-part2.sh
2020-05-25T04:20:05.5974901Z   SSH_ACTIONS: false
2020-05-25T04:20:05.5975013Z   UPLOAD_BIN_DIR: false
2020-05-25T04:20:05.5975102Z   UPLOAD_FIRMWARE: true
2020-05-25T04:20:05.5975208Z   UPLOAD_COWTRANSFER: false
2020-05-25T04:20:05.5975315Z   UPLOAD_WETRANSFER: false
2020-05-25T04:20:05.5975420Z   TZ: Asia/Shanghai
2020-05-25T04:20:05.5975547Z   FIRMWARE: /home/runner/work/3800test2/3800test2/openwrt/bin/targets/ar71xx/generic
2020-05-25T04:20:05.5975672Z ##[endgroup]
2020-05-25T04:20:05.9679183Z With the provided path, there will be 1 files uploaded
2020-05-25T04:20:06.6797068Z Total size of all the files uploaded is 847 bytes
2020-05-25T04:20:06.7167152Z Finished uploading artifact OpenWrt_firmware. Reported size is 847 bytes. There were 0 items that failed to upload
2020-05-25T04:20:06.7167647Z Artifact OpenWrt_firmware has been successfully uploaded!
2020-05-25T04:20:06.7303867Z Post job cleanup.
2020-05-25T04:20:06.8774988Z [command]/usr/bin/git version
2020-05-25T04:20:06.8836605Z git version 2.26.2
2020-05-25T04:20:06.8884514Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-05-25T04:20:06.8936578Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-05-25T04:20:06.9382619Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-05-25T04:20:06.9414477Z http.https://github.com/.extraheader
2020-05-25T04:20:06.9423620Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2020-05-25T04:20:06.9465198Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-05-25T04:20:06.9813696Z Cleaning up orphan processes

--- a/target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wdr8500-v1.c
+++ b/target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wdr8500-v1.c
@@ -9,6 +9,7 @@
  */
 
 #include <linux/gpio.h>
+#include <linux/phy.h>
 #include <linux/platform_device.h>
 #include <linux/proc_fs.h>
 #include <linux/spi/spi_gpio.h>
@@ -105,7 +106,7 @@ static struct ar8327_platform_data TL_WDR8500_v1_ar8337_data = {
 static struct mdio_board_info TL_WDR8500_v1_mdio0_info[] = {
 	{
 		.bus_id = "ag71xx-mdio.0",
-		.phy_addr = 0,
+		.mdio_addr = 0,
 		.platform_data = &TL_WDR8500_v1_ar8337_data,
 	},
 };

